Packs/DSPM/Integrations/DSPM/DSPM.yml

```
@@ -0,0 +1,984 @@
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'AWS'
+      - 'AZURE'
+      - 'GCP'
+      - 'SNOWFLAKE'
+      - 'FILE_SHARE'
+      - 'O365'
+    - description: 'Exact cloud provider ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"].'
+      isArray: false
+      name: cloud_provider_equals
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'AWS'
+      - 'AZURE'
+      - 'GCP'
+      - 'SNOWFLAKE'
+      - 'FILE_SHARE'
+      - 'O365'
+    - description: 'List of destination project vendor names.'
+      isArray: true
+      name: destination_project_vendor_name_in
+      required: false
+      secret: false
+    - description: 'Exact destination project vendor name.'
+      isArray: false
+      name: destination_project_vendor_name_equals
+      required: false
+      secret: false
+    - description: 'List of cloud environments ["UNKNOWN", "DEVELOPMENT", "STAGING", "TESTING", "PRODUCTION"].'
+      isArray: true
+      name: cloud_environment_in
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'UNKNOWN'
+      - 'DEVELOPMENT'
+      - 'STAGING'
+      - 'TESTING'
+      - 'PRODUCTION'
+    - description: 'Exact cloud environment ["UNKNOWN", "DEVELOPMENT", "STAGING", "TESTING", "PRODUCTION"].'
+      isArray: false
+      name: cloud_environment_equals
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'UNKNOWN'
+      - 'DEVELOPMENT'
+      - 'STAGING'
+      - 'TESTING'
+      - 'PRODUCTION'
+    - description: 'List of policy severities ["HIGH", "MEDIUM", "LOW"].'
+      isArray: true
+      name: policy_severity_in
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'HIGH'
+      - 'MEDIUM'
+      - 'LOW'
+    - description: 'Exact policy severity ["HIGH", "MEDIUM", "LOW"].'
+      isArray: false
+      name: policy_severity_equals
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'HIGH'
+      - 'MEDIUM'
+      - 'LOW'
+    - description: 'List of category types ["FIRST_MOVE", "ATTACK", "COMPLIANCE", "ASSET_AT_RISK", "RECONNAISSANCE"].'
+      isArray: true
+      name: category_type_in
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'FIRST_MOVE'
+      - 'ATTACK'
+      - 'COMPLIANCE'
+      - 'ASSET_AT_RISK'
+      - 'RECONNAISSANCE'
+    - description: 'Exact category type ["FIRST_MOVE", "ATTACK", "COMPLIANCE", "ASSET_AT_RISK", "RECONNAISSANCE"].'
+      isArray: false
+      name: category_type_equals
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'FIRST_MOVE'
+      - 'ATTACK'
+      - 'COMPLIANCE'
+      - 'ASSET_AT_RISK'
+      - 'RECONNAISSANCE'
+    - description: 'List of statuses ["OPEN", "UNIMPORTANT", "WRONG", "HANDLED", "INVESTIGATING"].'
```

```suggestion
    - description: 'List of statuses'
```
Is this a comma-separated list?  If so, it should be written as:
A comma-separated list of statuses.